usagov/usagov-2021
# USAGOV-1815: add hover affect to CTA button on homepage

## Jira Task

[USAGOV-1815](https://cm-jira.usa.gov/browse/USAGOV-1813)

## Description


Adding a hover affect to the CTA button located on the home page. 

## Type of Changes

![Screenshot 2024-08-07 at 3 07 13 PM](https://github.com/user-attachments/assets/286cc1ab-d428-4b54-b394-8bffcd323278)


## Testing Instructions



1. cd into the theme and rebuild to capture the new style changes
2. visit home page and scroll to the CTA for `Begin benefits search` and hover over button and verify that new style is applied 
3. also visit `/benefits` page and check that the same button has the hover affect styles working as expected

### Change Requirements



### Validation Steps

- check that hover is woking as expected and link is still directing user to correct page
## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
4. Build and deploy the changes.
5. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
